webdriver/tests/classic/perform_actions/key.py

```diff
@@ -24,10 +24,19 @@ def test_no_browsing_context(session, closed_frame, key_chain):
 def test_key_down_closes_browsing_context(
     session, configuration, http_new_tab, inline, key_chain
 ):
-    session.url = inline("""
+    url = inline("""
         <input onkeydown="window.close()">close</input>
         <script>document.querySelector("input").focus();</script>
         """)
+
+    # Open a new window.
+    new_window = session.execute_script(f"return window.open('{url}')")
+
+    new_window_handle = new_window.id
+
+    # Switch to the new window.
+    session.window_handle = new_window_handle
```

Comment:
Given that we use the variable only once lets get rid of it (applies to the other files as well). Also with the rename it's clear enough.

```suggestion
    # Switch to the new window.
    session.window_handle = new_window.id
```

Reply:
done